### PR TITLE
Feature romanian language

### PR DESCRIPTION
**Pull request by baakaweb**
## Romanian language

Adds Romanian language to the app

### CODE REVIEW

File: app/frontend/startup.js [hunk 1]
```diff
@@ -29,8 +29,6 @@ const i18nController = require('./controllers/i18n_controller.js');
 const UiHelper = require('./helpers/ui_helper.js');
 window.uiHelper = new UiHelper();
 
-const { waitUntilIdle } = require('./helpers/ezra_helper.js');
-
```
Comment (tobias-klein) [inline app/frontend/startup.js:33]: Not sure why these two lines were deleted, maybe by accident? Please restore.

File: html/bible_browser_toolbox.html [hunk 2]
```diff
@@ -41,9 +41,7 @@ <h3 class="ui-accordion-header ui-helper-reset ui-state-default ui-state-active
 </div>
 
 <div id="dictionary-info-box" class="accordion-panel">
-  <h3>
-    <a href="#" id="dictionary-info-box-header" i18n="[html]dictionary-info-box.default-header"></a>
-  </h3>
```
Comment (tobias-klein) [inline html/bible_browser_toolbox.html:46]: Not sure why these changes were introduced, please restore.

**Review by tobias-klein — CHANGES_REQUESTED** [review]
@baakaweb Thank you so much for contributing a new translation! Very much appreciated! You seem to have accidentally introduced some additional changes in app/startup.js and html/bible_browser_toolbox.html - please recover those.

File: app/frontend/startup.js (hunk 1 above)
Comment (baakaweb) [reply app/frontend/startup.js:33]: yes from the vs code ... when i had the conflict... I will revert it.

File: html/bible_browser_toolbox.html (hunk 2 above)
Comment (baakaweb) [reply html/bible_browser_toolbox.html:46]: very strange i did not touched that one :( Maybe at rebase. 
Let me  close this one and do it again as i have the changes locally
